src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1744,6 +1744,8 @@
   "block.minecolonies.blockhutmechanic": "Mechanic's Hut",
 
   "com.minecolonies.coremod.gui.workerhuts.plantation": "Plantation",
+  "com.minecolonies.coremod.gui.workerhuts.plantation.to_plant": "To plant:",
+  "com.minecolonies.coremod.gui.workerhuts.plantation.not_plant": "To NOT plant:",
```

Comment:
Maybe do "Planting:" and "Not planting:" here?

Reply:
Well neither here nor there, but I suppose the present continuous tense does make sense.